config/settings/development.yml

```diff
@@ -5,3 +5,4 @@ mcbg:
 system_authentication_token: "Ge32"
 log_level: debug
 gcp_api_key: please_change_me
+developer_auth: false
```

Comment:
no need for this change...

you can over override with `development.local.yml`

Reply:
Good spot. I wasn't intending to add this file.